velox/substrait/VeloxToSubstraitExpr.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include "velox/substrait/proto/substrait/algebra.pb.h"
+
+#include "core/PlanNode.h"
+
+#include "VeloxToSubstraitFunc.h"
+#include "VeloxToSubstraitType.h"
+
+using namespace facebook::velox::core;
+
+namespace facebook::velox::substrait {
+
+class VeloxToSubstraitExprConvertor {
+ public:
+  void transformVExpr(
```

Comment:
maybe `transformVeloxExpr()` to make it more explicit?

Reply:
sure, fixed.